src/index.html

```diff
@@ -1,16 +1,28 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <div class="form-field" data-qa="big">
+        <input class="bigsearch" type="search" 
+        name="firstsearch" data-qa="keypress" 
+        placeholder="Try &quot;Los Angeles&quot;">
+      </div>
+    </form>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <div class="form-field" data-qa="small">
+        <input class="smallsearch" type="search" 
+        name="secondsearch" placeholder="Try &quot;Los Angeles&quot;">
+      </div>
+    </form>
```

Comment:
https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md
Check checklist
```suggestion
    </form>
    
    <form
       action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
       method="post"
     >
      <div class="form-field" data-qa="small">
        <input 
          class="smallsearch" 
          type="search" 
          name="secondsearch" 
          placeholder="Try &quot;Los Angeles&quot;"
        >
      </div>
    </form>
```